Sources/PlatformVersion.swift

```diff
@@ -238,7 +240,9 @@ extension tvOSVersion {
         if #available(tvOS 27, *) {
             return .past
         }
-        if #available(tvOS 26, *) {
+        // Apps built before the tvOS 26 SDK get "19.0" as the system version from ProcessInfo.
+        // Once built with the tvOS 26 SDK, the version then becomes "26.0".
+        if #available(tvOS 19, *) {
```

Comment:
I don't know this for sure, I'm just assuming based on what the iOS SDK does. In any case it's safe to do as tvOS 19 will never actually exist.